src/Main.bas

```diff
@@ -6827,7 +6818,7 @@ Function CheckCompilerPaths As Boolean
 	#endif
 	If Not bFind Then
 		If MsgBox(ML("Invalid defined compiler path.") & !"\r" & ML("Find Compilers from Computer?"), , mtQuestion, btYesNo) = mrYes Then
-			pfOptions->Show *pfrmMain
+			pfOptions->ShowModal *pfrmMain
```

Comment:
If it not use showModal. About 3 forms will be showing in the same time.

Reply:
Then it needs to be done in the OnShow event of the form. Because, after ShowModal, the next two lines are not executed. The user can only see the Options form. All actions begin after the closed form. Please correct this.